embedded-game-controller/client.h

```diff
@@ -0,0 +1,10 @@
+#ifndef EGC_CLIENT_H
+#define EGC_CLIENT_H
+
+#include <stdbool.h>
+
+#define MAX_FAKE_WIIMOTES 2
+
+void egc_set_sensor_bar_position_top(bool on_top);
+
```

Comment:
I think this should be moved to Fakemote-specific code, but we can do that later. For now, let's merge this PR to move forward!

Reply:
OK, I'll merge it then, thanks!